docs/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!--- If applicable, issue number goes here, e.g. GODRIVER-ABCD -->
```

Comment:
Optional: Consider including a template for a link to a GODRIVER issue since it's very common to include a link.
```
[GODRIVER-0000](https://jira.mongodb.org/browse/GODRIVER-0000)
```

Reply:
Is this template also applied to external contributors? If so, they may not have a ticket number.